tag-security-questionnaire.md

```diff
@@ -3,7 +3,8 @@
 
 The Storage Access API enables the removal of cross-site cookies. Specifically, it allows the authenticated embeds use case to continue to work. As such, the API provides a way for developers to re-gain access to cross-site cookies, albeit under further constraints.
 
-A nested Document gains access to the same cookies it has as the active document of a top-level browsing context when it calls `document.requestStorageAccess()` and is returned a resolving Promise. With these cookies it can authenticate itself to the server and load user-specific information.
+A nested Document gains access to the same cookies it has as the active document of a top-level browsing context when it calls `document.requestStorageAccess()` and is returned a resolving Promise (while this doesn't change the threat model, `SameSite` restrictions still apply; in practice, only `SameSite=None` cookies should be granted).
```

Comment:
Hmm I'm not a fan of "in practice" here, slightly reworded suggestion:

`SameSite restrictions will still apply, i.e. SameSite=Lax or Strict cookies will not be available, which is important for security but doesn't affect the privacy considerations)`

This sentence most notably doesn't care what the default state in a particular browser is.

Thoughts? :)

Reply:
updated it with this wording basically verbatim